doc/rtd/reference/custom_modules/custom_mergers.rst

```diff
@@ -0,0 +1,66 @@
+.. _custom_mergers:
+
+Custom Mergers
+**************
+
+TODO adapt and backreference
```

Comment:
Is this TODO for this PR or for later? If for later, let's remove it before merging.

Reply:
Removed.